assets/section-footer.css

```diff
@@ -302,12 +301,34 @@
 }
 
 .localization-form__select {
-  box-shadow: 0 0 0 0.1rem rgba(var(--color-foreground), 0.55);
+  border-radius: calc(var(--inputs-border-width) + var(--inputs-radius));
   position: relative;
   margin-bottom: 1.5rem;
-  border: 0.1rem solid transparent;
   padding-left: 1rem;
   text-align: left;
+  min-height: calc(4rem + var(--inputs-border-width) * 2);
+  min-width: calc(7rem + var(--inputs-border-width) * 2);
+}
+
+.disclosure__button.localization-form__select {
+  padding: calc(1.5rem + var(--inputs-border-width))
+}
+
+.localization-form__select:before {
+  content: '';
+  position: absolute;
+  inset: 0;
+  border-radius: calc(var(--inputs-border-width) + var(--inputs-radius));
```

Comment:
I just noticed that this technique can lead to unnecessary radius even when it's 0 (because of the border value).
 <img src="https://screenshot.click/10-16-y5qrc-eguzw.png" width="350"/> 
 
I know this isn't going to be unique to inputs though so we don't need to deal with it here, but we'll have to decide how we want to handle that. Something contrived like `calc(var(--inputs-radius) * var(--inputs-radius) + var(--inputs-border-width) + var(--inputs-radius))` could work inline, but we may consider adding additional global vars like `--inputs-radius-outset: {% if settings.radius > 0 %} radius + border_width {% else %) 0 {%endif%}`

Reply:
Good catch!